cluster/ubuntu/util.sh

```diff
@@ -450,7 +450,61 @@ function kube-down {
 
 # Update a kubernetes cluster with latest source
 function kube-push {
-  echo "not implemented"
+  #stop all the kube's process & etcd 
+  KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
+  source "${KUBE_ROOT}/cluster/ubuntu/${KUBE_CONFIG_FILE-"config-default.sh"}"
+
+  for i in ${nodes}; do
+    {
+      echo "Cleaning on node ${i#*@}"
+      ssh -t $i 'sudo -p "[sudo] stop the all process: " service etcd stop'
+      ssh -t $i 'rm -f /opt/bin/kube* /etc/init/kube* /etc/init.d/kube* /etc/default/kube*; rm -rf ~/kube'
+    }
+  done
+  wait
+
+  # The latest version tag of kubernetes can be got by gsutil, or set in config-default.sh 
+  K8S_VERSION=$(gsutil cat gs://kubernetes-release/release/latest.txt)
```

Comment:
This is already set here: https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-push.sh#L65. You should use that one so that the kube-push usage is correct